components/script/dom/webglbuffer.rs

```diff
@@ -113,6 +137,15 @@ impl WebGLBuffer {
         self.is_deleted.get()
     }
 
+    pub fn is_currently_bound(&self) -> bool {
+        self.currently_bound.get()
+    }
+
+    pub fn set_currently_bound(&self, value: bool) {
```

Comment:
I don't think this quite works.  Bind a buffer to COPY_READ, then COPY_WRITE, then unbind from COPY_WRITE, and now it's "not currently bound" and we would allow it to be bound to TFB, right?